packages/api/src/utils/generate_id.rs

```diff
@@ -3,7 +3,7 @@ use time::OffsetDateTime;
 
 // prefix_[48 bit timestamp][80 bit random payload]
 const TIMESTAMP_BYTES: usize = 6; // 48 bits for timestamp
-const TOTAL_BYTES: usize = 16; // 128 bits total
+const TOTAL_BYTES: usize = 20; // 128 bits total
```

Comment:
```suggestion
const TOTAL_BYTES: usize = 20; // 160 bits total
```

idk going back and forth on this